info.md

```diff
@@ -5,171 +5,177 @@
 ![logo](https://github.com/rospogrigio/localtuya-homeassistant/blob/master/img/logo-small.png)
 
 A Home Assistant custom Integration for local handling of Tuya-based devices.
-Device status is updated receiving push updates from the device instead of polling, so status updates are extremely fast (even if manually operated).
+
+This custom integration updates device status via push updates instead of polling, so status updates are fast (even when manually operated).
```

Comment:
```suggestion
This custom integration updates device status via pushing updates instead of polling, so status updates are fast (even when manually operated).
```